### ISSUE TITLE
Standalone python build

### ISSUE THREAD

**Issue by FrancescoCeruti** [OpenLightingProject/ola#1200]
Lately I'm trying to bundle an application into an AppImage, and I've noticed that the OLA python client API is independent from the rest of the compiled modules (right?).

Since ola is just used for powering one plugin, my objective is to bundle only the python client and leave the task of installing ola (the daemon) to the user via the distribution packages.
This is possible by coping manually only the "ola" python package from the compilation result but I need to run the whole process to get the `protobuf` modules.

Is possible to build the only the python modules?

**Comment by peternewman** [comment]
I think you should be able to avoid most of it. This is what we run for flake8 and pychecker checking of our Python code, so I believe it makes all the Python files we need:
https://github.com/OpenLightingProject/ola/blob/master/.travis-ci.sh#L99

Debian and co ship ola-python if that's of use, or you can make use of how their building, but I imagine that may build everything too.

**Comment by FrancescoCeruti** [comment]
Thanks, that works, and it's a lot quicker :)

Anyway, I believe it would be handy to be able to `configure` to build just one, or more, of the clients API in such situations.

**Comment by peternewman** [comment]
Okay @FrancescoCeruti I've changed it into an enhancement request, so you'd like to do the normal configure, make cycle and have only the python API being built? Or the C++ one, or both. So essentially it's almost a way to disable building the majority of OLA.

Doing the Python one would be fairly easy as you've seen. Unpicking the requirements for the C++ client from the rest of OLA may be rather more challenging...

**Comment by FrancescoCeruti** [comment]
@peternewman Yeah, that's the idea, I personally need only the python API, but maybe someone can find  it useful for similar situations :)
